services/web/server/src/simcore_service_webserver/computation_subscribe.py

```diff
@@ -34,45 +36,73 @@
-    # update corresponding project, node, progress value
-    rabbit_message = ProgressRabbitMessage.parse_raw(data)
+async def _handle_computation_running_progress(
+    app: web.Application, message: ProgressRabbitMessage
+) -> bool:
     try:
         project = await projects_api.update_project_node_progress(
             app,
-            rabbit_message.user_id,
-            f"{rabbit_message.project_id}",
-            f"{rabbit_message.node_id}",
-            progress=rabbit_message.progress,
+            message.user_id,
+            f"{message.project_id}",
+            f"{message.node_id}",
+            progress=message.progress,
         )
         if project:
             messages: list[SocketMessageDict] = [
                 {
                     "event_type": SOCKET_IO_NODE_UPDATED_EVENT,
                     "data": {
-                        "project_id": project["uuid"],
-                        "node_id": rabbit_message.node_id,
-                        "data": project["workbench"][f"{rabbit_message.node_id}"],
+                        "project_id": message.project_id,
+                        "node_id": message.node_id,
+                        "data": project["workbench"][f"{message.node_id}"],
                     },
                 }
             ]
-            await send_messages(app, f"{rabbit_message.user_id}", messages)
+            await send_messages(app, f"{message.user_id}", messages)
             return True
     except ProjectNotFoundError:
         log.warning(
             "project related to received rabbitMQ progress message not found: '%s'",
-            json_dumps(rabbit_message, indent=2),
+            json_dumps(message, indent=2),
         )
         return True
     except NodeNotFoundError:
         log.warning(
             "node related to received rabbitMQ progress message not found: '%s'",
-            json_dumps(rabbit_message, indent=2),
+            json_dumps(message, indent=2),
         )
         return True
     return False
 
 
+async def progress_message_parser(app: web.Application, data: bytes) -> bool:
+    # update corresponding project, node, progress value
+    rabbit_message = ProgressRabbitMessage.parse_raw(data)
+
+    if rabbit_message.progress_type is ProgressType.COMPUTATION_RUNNING:
+        # NOTE: backward compatibility, this progress is kept in the project
+        return await _handle_computation_running_progress(app, rabbit_message)
+
+    # NOTE: other types of progress are transient
+    await send_messages(
+        app,
+        f"{rabbit_message.user_id}",
+        [
+            {
+                "event_type": SOCKET_IO_NODE_PROGRESS_EVENT,
+                "data": {
+                    "project_id": rabbit_message.project_id,
+                    "node_id": rabbit_message.node_id,
+                    "user_id": rabbit_message.user_id,
+                    "progress_type": rabbit_message.progress_type.name,
```

Comment:
with the modification i suggest above, you will  not need to pass ".name" here

Reply:
yeah